packages/button/src/__demo__/examples/circular.js

```diff
@@ -29,20 +29,22 @@ export default {
   title: 'Circular Buttons',
   description:
     'Buttons can be made circular. Such Buttons should not have any text.',
```

Comment:
While I think we should still advise that circular Buttons should not have text content, they now _can_, because it no longer requires a fixed-width.